pkg/backup/backup/backup_manager.go

```diff
@@ -72,7 +74,67 @@ func (bm *backupManager) syncBackupJob(backup *v1alpha1.Backup) error {
 	ns := backup.GetNamespace()
 	name := backup.GetName()
 	backupJobName := backup.GetBackupJobName()
+	logBackupSubcommand := v1alpha1.ParseLogBackupSubcommand(backup)
+	var err error
+
+	// validate backup
+	if err = bm.validateBackup(backup); err != nil {
+		klog.Errorf("backup %s/%s validate error %v.", ns, name, err)
+		return err
+	}
+
+	// skip backup
+	skip := false
+	if skip, err = bm.skipBackupSync(backup); err != nil {
+		klog.Errorf("backup %s/%s skip error %v.", ns, name, err)
+		return err
+	} else {
```

Comment:
you can remove the `else` to make codes more concise

Reply:
ok, i will fix the codes like this.